### PR TITLE
Update board_recruiting.vue

### PR DESCRIPTION
**Pull request by SteveReiter**
Fixing spelling and grammar on the board recruiting page.

### CODE REVIEW

File: pages/board_recruiting.vue [hunk 1]
```diff
@@ -15,9 +15,9 @@
                 </h2>
                 <p class="py-2">
                     The Dev Edmonton Board consists of 7 members filling three
-                    year terms. Every year at the annual meeting elctions will
-                    be held for any open positions. Dev Edmonton Society is
-                    desires to build a board with a diversity of perspectives
+                    year terms. Every year at the annual meeting elections will
+                    be held for any open positions. Dev Edmonton Society
+                    desires to build a board with diverse perspectives
```
Comment (SteveReiter) [inline pages/board_recruiting.vue:20]: I just think "diverse perspectives" reads easier than "a diversity of perspectives" but I think the original is still fine. I can remove that change if you want.

Comment (MarkBennett) [reply pages/board_recruiting.vue:20]: Nope, that reads better to me. Merging. Thanks!

**Comment by MarkBennett** [comment]
@allcontributors add @SteveReiter as documentation

**Comment by MarkBennett** [comment]
@allcontributors add @SteveReiter as documentation

**Pull request merged**